evap/staff/forms.py

```diff
@@ -702,6 +703,10 @@ def save(self, *args, **kw):
 
         self.instance.is_active = not self.cleaned_data.get('is_inactive')
 
+        # refresh results cache
+        for evaluation in Evaluation.objects.filter(contributions__contributor=self.instance).distinct():
+            collect_results(evaluation, force_recalculation=True)
```

Comment:
as this can take a while on some userprofiles, it might be worth it to restrict this to `if any(attribute in self.changed_data for attribute in ['first_name', 'last_name', 'title']` (untested).